R/utils.R

```diff
@@ -115,6 +115,26 @@ clean_vrd_api <- function(api_output){
     # adds back the correct column names
     colnames(df3) <- df1$Name
 
+    # checks column types
+    col_type <- df1[, grep("Type", colnames(df1))]
+
+    # assigns columns
+    i <- 1
+    for (element in col_type[[1]]){
+
+        # change df type
+        if (element == "System.Decimal"){
+            df3[[i]] <- as.integer(df3[[i]])
```

Comment:
Check if its a decimal type then casting to integer seems a bit strange

Reply:
Its true, but I can't think of a single column this database outputs that has decimals.  And having years as decimals seems weird.

Reply:
alright

👍